### DISCUSSION TITLE
Multi Database Serenity

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5358] [Help]
![image](https://user-images.githubusercontent.com/24252615/104881287-177ace00-5972-11eb-9f92-fb285e5d690a.png)
![image](https://user-images.githubusercontent.com/24252615/104881342-324d4280-5972-11eb-9fbf-53e009c9156b.png)

I'm trying to make multiple databases, but I'm getting an error like this. Where could I have made an error?

**Comment by edwardch** [comment]
Has `UserRetrieveService` been registered? Check in `ConfigureServices` in Startup.cs.
